dvc/command/gc.py

```diff
@@ -74,6 +74,8 @@ def add_parser(subparsers, parent_parser):
     gc_parser.add_argument(
         "-r", "--remote", help="Remote repository to collect garbage in."
     )
+    # What does this actually mean? Tracing the code, it seems the only place this parameter is used is when deciding
```

Comment:
Correct, it is used there and also in `dvc/command/gc.py` to confirm that you want to proceed with gc.

Reply:
Basically what it says is "don't ask questions, just clean up garbage" :slightly_smiling_face: I hope this makes some sense.

Reply:
As @shcheklein noted, it might not be clear why Repo._collect_dir_cache has a prompt. So the way we store cache for directories is as follows:

1) `checksum.dir` file under .dvc/cache that is actually a json file that contains list of all files from that directory along with checksums for those files;
2)  for each file from that directory we store `checksum` cache file under .dvc/cache'

So theoretically there might be a situation where your data directory doesn't have that `checksum.dir` file for it for some reason. Without it, dvc can't account for all cache files inside of that dir and thus you run the risk of losing them. With `--force`, dvc won't ask confirmation that you are sure you want to proceed in that situation.

Reply:
@efiop Thanks for the explanation, I failed to notice the reference to `force` in the gc prompt.

Regarding the cache dirs, I actually figured that one out. I was just saying that as an explanation for what this flag does, it's not helpful, and I'm not sure how to clarify it. I think `force` is too general. Maybe something like `-y --yes Automatically agree to all prompts` better explains what's going on here.

Reply:
@efiop thanks for the explanation! we should update the doc with this info. 

@guysmoilov may be we can keep it `-f` but add your copy `Automatically agree to all prompts`?

Reply:
@shcheklein I added the explanation to the -f flag

Reply:
@guysmoilov thanks!